examples/surface_timeseries.py

```diff
@@ -0,0 +1,33 @@
+"""
+Display a surface timeseries using data from nilearn
+"""
+
+try:
+    from nilearn import datasets
+    from nilearn import surface
+except ImportError:
+    raise ImportError("""This example uses data and methods from nilearn but
+    nilearn installed. To install try 'pip install nilearn'.""")
+
+import numpy as np
+import napari
+
+
+# Fetch datasets - this will download dataset if datasets are not found
+nki_dataset = datasets.fetch_surf_nki_enhanced(n_subjects=1)
+fsaverage = datasets.fetch_surf_fsaverage()
+
+# Load surface data and resting state time series from nilearn
+pial_left = surface.load_surf_data(fsaverage['pial_left'])
+sulc_left = surface.load_surf_data(fsaverage['sulc_left'])
+timeseries = surface.load_surf_data(nki_dataset['func_left'][0]).transpose((1, 0))
```

Comment:
I *slightly* would prefer that we transpose when adding the data, so that users can see in the call itself that our format is transposed from nilearn's. Or, at least, put the transpose in its own line. You could also add a comment on top of the line with the transpose, "nilearn provides data as n_vertices x n_timepoints (x n_trials?), but napari requires the vertices axis to be placed last to match NumPy broadcasting rules."

Reply:
done